src/entities/course/types.ts

```diff
@@ -15,3 +15,14 @@ export type Course = {
     accentColor: string;
   };
 };
+
+export type CourseCardProps = Pick<
+  Course,
+  'title' | 'iconSrc' | 'startDate' | 'detailsUrl' | 'mode' | 'language' | 'backgroundStyle'
+>;
+
+export enum CourseStatus {
```

Comment:
```suggestion
export const enum CourseStatus {
```

Reply:
Interesting, didn't heard of this issue before. Personally I have an allergy for enums and prefer to avoid them in favor of objects or union types. You can discuss on  best approach in this case with team or decide on your own. Feel free to resolve this conversation.

Reply:
I prefer union-types too. I'll change it